frontend/catalyst/pipelines.py

```diff
@@ -74,6 +74,8 @@ class CompileOptions:
             Default is None.
         pass_plugins (Optional[Set[Path]]): List of paths to pass plugins.
         dialect_plugins (Optional[Set[Path]]): List of paths to dialect plugins.
+        track_resources (Optional[bool]): Flag indicating whether to enable experimental resource
+            tracking (NullQubit only).
```

Comment:
Does it make sense as a QJIT parameter if it applies to NullQubit only? We also don't seem to make use of this option anyway.

Reply:
Good catch, not sure how I ended up adding this. Removed in [1cda33a](https://github.com/PennyLaneAI/catalyst/pull/1619/commits/1cda33a3d4b4960b1b5cffc329278e1b1ef1b1c1)